tequilapi/endpoints/connection_test.go

```diff
@@ -76,7 +84,11 @@ func TestAddRoutesForConnectionAddsRoutes(t *testing.T) {
 		},
 		{
 			http.MethodGet, "/connection/statistics", "",
-			http.StatusOK, `{"bytesSent": 0, "bytesReceived": 0}`,
+			http.StatusOK, `{
+				"bytesSent": 0,
+				"bytesReceived": 0,
+				"durationSeconds": 60
```

Comment:
what about just "sessionDuration"? It can be just convention - sent,received are reported as bytes, duration is reported as seconds. Api consumer when can reformat to any precision needed.

Reply:
`bytesSent` and `bytesReceived` specifies that it's `bytes` in the name, while `duration` does not. At first I implemented it in `duration` myself, but having raw number without explanation about value seemed confusing - also` durationSeconds` is more consistent with `bytesSent`/`bytesReceived`.

Reply:
In that case: maybe sentBytes, receivedBytes and sessionSeconds? :) or vice versa. More consistent